docs/advanced/app-connectors.md

```diff
@@ -10,13 +10,16 @@ The connectors are composed of a root project which implements a base connector
 
 ### The connector
```

Comment:
Should we add as an intro that we are describing the structure that we use in our own connectors (and that we recommend to follow)?